lib/os-cluster-entrypoint.ts

```diff
@@ -136,8 +136,14 @@ export class OsClusterEntrypoint {
 
       this.stacks.push(network);
 
+      if (suffix === 'undefined') {
+        infraStackName = 'OpenSearch-Infra-Stack';
+      } else {
+        infraStackName = `OpenSearch-Infra-Stack-${suffix}`;
```

Comment:
Nit: opensearch-cluster-stack , (keep it all small just for convenience and avoiding complexity when it comes to integrating with other systems)

Reply:
Ack!